MyHomework #2

OpalevAndrey opened the pull request:
Ссылка на элемент с сайта СКБ-Контура.
http://www.skbkontur.ru/Files/Modules/Product/30_l.jpg
Не уверен, что правильно, т.к. пропустил занятие. Просьба, если будет
время и желание, высказать замечания.

mokhov commented:
Не принято.
Нужно придумать максимальное число разных селекторов, которыми можно выбрать этот элемент с сайта Контура. Творчество приветствуется

OpalevAndrey committed:
Домашнее задание по CSS.

Ссылка на элемент с сайта СКБ-Контура.
http://www.skbkontur.ru/Files/Modules/Product/30_l.jpg
Не уверен, что правильно, т.к. пропустил занятие. Просьба, если будет
время и желание, высказать замечания.
